pkg/divert/weaver/cache_test.go

```diff
@@ -86,7 +86,7 @@ func Test_initCache(t *testing.T) {
 		},
 	}
 
-	d := &Driver{client: c, manifest: m}
+	d := &Driver{client: c, name: m.Name, namespace: m.Namespace, divert: *m.Deploy.Divert}
```

Comment:
@jLopezbarb @AgustinRamiroDiaz @ifbyol I followed the same approach as the Istio driver and remove the pointer to the manifest and copy the needed fields instead.